### DISCUSSION TITLE
Website is broken

### DISCUSSION THREAD

**Discussion by tormgibbs** [ueberdosis/tiptap#5313] [General Questions]
I have tried to open the website in Chrome and edge but it doesnt go through. It'll randomly come and break again when i click on a link

**Comment by nperez0111** [comment]
Is it still an issue?

**Reply by discoverlance-com** [reply]
Website is down for me here. It's not opening for me at all...

I Just get this error


```
This site can’t be reached
The connection was reset.
Try:

Checking the connection
...
ERR_CONNECTION_RESET

```
